app/src/main/assets/buildingcoordinates.json

```diff
@@ -0,0 +1,1541 @@
+{
+  "type": "FeatureCollection",
```

Comment:
do we need this file?

Reply:
It was used before for the polygons.. not sure if we still need it